app/src/pages/inside/dashboardPage/dashboardList/dashboardEmptyResults/dashboardEmptyResults.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { GhostButton } from 'components/buttons/ghostButton';
+import styles from './dashboardEmptyResults.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  currentUserDashboardsHeadline: {
+    id: 'DashboardEmptyResults.currentUserDashboardsHeadline',
+    defaultMessage: 'You have no dashboards',
+  },
+  currentUserDashboardsText: {
+    id: 'DashboardEmptyResults.currentUserDashboardsText',
+    defaultMessage: 'Add your first dashboard to analyse statistics',
+  },
+  currentUserDashboardsActionText: {
+    id: 'DashboardEmptyResults.currentUserDashboardsActionText',
+    defaultMessage: 'Add New Dashboard',
+  },
+  sharedDashboardsHeadline: {
+    id: 'DashboardEmptyResults.sharedDashboardsHeadline',
+    defaultMessage: 'No dashboards are shared',
+  },
+});
+
+@injectIntl
+export class DashboardEmptyResults extends Component {
```

Comment:
Maybe NoDashboardsView or EmptyDashboard is better name, "DashboardEmptyResults" sounds like you have entered into dashboard and it's not contains any widgets.